src/app/features/tracking/tracking.component.ts

```diff
@@ -37,9 +48,22 @@ export class TrackingComponent implements OnInit {
       this.products = products;
     });
   }
-  getProduct(id: string) {
+  getProduct(id: ObjectId) {
     this.productService.getProductById(id).subscribe((product: Product) => {
       this.product = product;
     });
   }
+
+  openSidebar(logis: Logistic) {
+    this.showDetails = true;
+    this.details = 'sidebar_details';
+    this.backgrond = 'show_backgrond';
+
+    const data = logis; // Dados a serem passados
```

Comment:
Change comment to english
```suggestion
    const data = logis; // Data to be sent
```